### ISSUE TITLE
No title (h1) on the page

### ISSUE THREAD

**Issue by selikem** [soniasophia/aloha-website#4]
A web page should have a title/heading that describes the topic or purpose of the page. This benefits assistive technologies, like screen readers, which can use headings to navigate a page.
